### PR TITLE
feat: Create New Relay Page

### PR DESCRIPTION
**Pull request by yashpandit**
Add new Page - Create New Relay

Resolves: #24

# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Unit Test
- [ ] Integration Test

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by yashpandit** [comment]
create_new_relay_1

**Comment by souravdasslg** [comment]
@yashpandit In responsive mode, the "create relay page" text is not aligned to the center. Especially in the 320X426 ratio.

**Comment by yashpandit** [comment]
working on it

**Comment by yashpandit** [comment]
create_new_relay_mobile_1

**Comment by yashpandit** [comment]
create_new_relay_web_1

File: client/src/components/common/index.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
 import Header from './Header';
```
Comment (nimish-gupta) [inline client/src/components/common/index.js]:
```suggestion
export Header from './Header';
```

File: client/src/components/common/index.js [hunk 2]
```diff
@@ -1,3 +1,3 @@
...
 
-export { Header };
+export default Header;
```
Comment (nimish-gupta) [inline client/src/components/common/index.js]: why default?

File: client/src/components/Dashboard/CreateNewRelay/CreateNewRelay.js [hunk 3]
```diff
@@ -1,5 +1,85 @@
...
-const CreateNewRelay = () => <h1>CreateNewRelay</h1>;
+const mobileScreen = window.innerWidth <= 425;
+
+const CreateNewRelay = () => {
+  return (
+    <>
+      <Header />
+      <Container>
+        <Form>
+          <Row>
```
Comment (nimish-gupta) [inline client/src/components/Dashboard/CreateNewRelay/CreateNewRelay.js]: you dont need row component if you have single col, you can replace the Row/Col pair with div and then add classes to the div.

File: client/src/components/Dashboard/CreateNewRelay/CreateNewRelay.js [hunk 4]
```diff
@@ -1,5 +1,85 @@
...
+              </Input>
+            </Col>
+            <Col className="align-center mt-4">
+              {(mobileScreen)
+                ? <img src={ArrowDown} alt="arrow" width="30px" />
+                : <img src={ArrowRight} alt="arrow" width="30px" />}
+            </Col>
+            <Col sm="12" md="4">
+              <Label>Select App 2</Label>
+              <Input type="select" name="app2" id="appSelect2" required>
```
Comment (nimish-gupta) [inline client/src/components/Dashboard/CreateNewRelay/CreateNewRelay.js]: You can create a generic select component which will take name and options as props.

File: client/src/components/Dashboard/CreateNewRelay/CreateNewRelay.js [hunk 5]
```diff
@@ -1,5 +1,85 @@
...
+              <Input type="select" name="trigger1" id="triggerSelect1" required>
+                <option>1</option>
+                <option>2</option>
+                <option>3</option>
+                <option>4</option>
+                <option>5</option>
+              </Input>
+            </Col>
+            <Col className="align-center mt-4">
+              {(mobileScreen)
```
Comment (nimish-gupta) [inline client/src/components/Dashboard/CreateNewRelay/CreateNewRelay.js]: Good call :)

File: client/src/components/common/index.js (hunk 2 above)
Comment (yashpandit) [reply client/src/components/common/index.js]: changed it back to destructuring

File: client/src/components/Dashboard/CreateNewRelay/CreateNewRelay.js (hunk 4 above)
Comment (yashpandit) [reply client/src/components/Dashboard/CreateNewRelay/CreateNewRelay.js]: Done

**Pull request merged**